src/components/common/TxHistory/TxHistoryModal/TxHistoryModal.tsx

```diff
@@ -85,42 +86,47 @@ const TxHistoryModal = (): JSX.Element => {
             </Flex>
           </Flex.Item>
           {txs ? (
-            normalizeOperations(txs).map((op, index) => {
-              return (
-                <Flex.Item
-                  key={index}
-                  style={{
-                    borderBottom: '1px solid var(--ergo-default-border-color)',
-                  }}
-                >
-                  <Box transparent padding={[5, 0]}>
-                    <Flex justify="space-between" align="center">
-                      <Flex.Item style={{ width: '35%' }}>
-                        <InputOutputColumn
-                          type={op.type}
-                          x={op.assetX}
-                          y={op.assetY}
-                        />
-                      </Flex.Item>
-                      <Flex.Item style={{ width: '28%' }}>
-                        {op.timestamp}
-                      </Flex.Item>
-                      <Flex.Item style={{ width: '20%' }}>
-                        <TxTypeTag type={op.type} />
-                      </Flex.Item>
-                      <Flex.Item style={{ width: '16%' }}>
-                        <TxStatusTag status={op.status} />
-                      </Flex.Item>
-                      <Flex.Item style={{ width: '5%' }}>
-                        <OptionsButton type="text" placement="bottomLeft">
-                          {renderTxActionsMenu(op)}
-                        </OptionsButton>
-                      </Flex.Item>
-                    </Flex>
-                  </Box>
-                </Flex.Item>
-              );
-            })
+            normalizeOperations(txs)
+              .sort((a, b) => Number(a.timestamp) + Number(b.timestamp))
+              .map((op, index) => {
+                return (
+                  <Flex.Item
+                    key={index}
+                    style={{
+                      borderBottom:
+                        '1px solid var(--ergo-default-border-color)',
+                    }}
+                  >
+                    <Box transparent padding={[5, 0]}>
+                      <Flex justify="space-between" align="center">
+                        <Flex.Item style={{ width: '35%' }}>
+                          <InputOutputColumn
+                            type={op.type}
+                            x={op.assetX}
+                            y={op.assetY}
+                          />
+                        </Flex.Item>
+                        <Flex.Item style={{ width: '28%' }}>
+                          {getFormattedDate(op.timestamp)}
```

Comment:
I think, we can replace many utility fns with DateTimeView component with next api:
```
<DateTimeView value={} display="date | time | datetime" />
```
What do you think ?